app/javascript/src/reports.js

```diff
@@ -9,7 +9,7 @@ document.addEventListener('DOMContentLoaded', () => {
 const handleReportFormSubmit = (event) => {
   event.preventDefault()
 
-  const buttonText = event.target.value
+  const buttonText = event.target.innerHTML
 
   event.target.disabled = 'disabled'
   event.target.value = event.target.dataset.disableWith
```

Comment:
I think you'll need to change this to `event.target.innerHtml = event.target.dataset.disableWith`

Reply:
And on line 20 you'll need to do the same  `event.target.innerHTML`

Reply:
I see! It makes sense. Thanks for changing it for me 🙂